.github/workflows/prepare-release.yml

```diff
@@ -28,6 +31,22 @@ jobs:
         echo "New version: $NEW_VERSION"
         find . -type f -not -path '*/\.*' -not -path './CHANGELOG.md' -not -path './docs/package-lock.json' -not -path './RELEASING.md' -exec sed -i "s/$CURRENT_VERSION/$NEW_VERSION/g" {} +
 
+    - name: Setup scarb
+      uses: software-mansion/setup-scarb@v1
+      id: setup_scarb
+      with:
+        scarb-version: ${{ env.SCARB_VERSION }}
+
+    - name: Setup class_hash
+      uses: ericnordelo/setup-class-hash@v1
```

Comment:
Regarding tags, it's safer to use the commit hash instead e.g. `ericnordelo/setup-class-hash@abcd1234`. If a project is compromised, it's really easy to overwrite a tag with malicious code (`git tag -f`). The one drawback with using the hash is that it misses out on updates and patches (which may address vulnerabilities that have been found on that specific version)